### ISSUE TITLE
version number not reported properly in built docs

### ISSUE THREAD

**Issue by prjemian** [prjemian/hklpy2#4]
Locally, the version number is correct but online docs are missing the MAJOR part of the version.

### Local
![image](https://github.com/prjemian/hklpy2/assets/2279984/0df8e42a-64a1-4f4d-b2c8-11879b7c13f1)

```
full version: 0.0.9.dev2+ge9d9a4e
```

### Online
![image](https://github.com/prjemian/hklpy2/assets/2279984/b1cc50a5-1b65-436e-9a75-114e678402e9)

```
full version: 0.1.dev1+ge9d9a4e
```

**Comment by prjemian** [comment]
Refactor and stop using https://github.com/sphinx-notes/pages

**Comment by prjemian** [comment]
Resolved with 907f300 and a few previous.
